python/cuml/dask/common/dask_arr_utils.py

```diff
@@ -53,6 +68,31 @@ def _conv_df_to_sp(x):
     return cp.sparse.csr_matrix(cupy_ary)
 
 
+def _conv_to_sparse(arr):
```

Comment:
Naming convention seems inconsistent:
 `_conv_df_to_sp` vs. `_conv_to_sparse`... maybe make them `_conv_df_to_sparse` and `_conv_array_to_sparse`?